examples/src/main/python/sql/arrow.py

```diff
@@ -275,6 +275,27 @@ def merge_ordered(left: pd.DataFrame, right: pd.DataFrame) -> pd.DataFrame:
     # +--------+---+---+----+
 
 
+def arrow_python_udf_example(spark: SparkSession) -> None:
+    from pyspark.sql.functions import udf
+
+    @udf(returnType='int')  # A default, pickled Python UDF
+    def slen(s):  # type: ignore[no-untyped-def]
```

Comment:
Why don't you just type it and remove the `ignore`?

Reply:
We don't seem to have type hints for any examples of Python UDFs since they are not a must-have as Pandas UDF. Would you prefer to type it here?

Reply:
okie. let's just go ahead for now then.